sklearn/preprocessing/tests/test_common.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+
+
+from sklearn.datasets import load_iris
+
+from sklearn.model_selection import train_test_split
+
+from sklearn.preprocessing import QuantileTransformer
+
+from sklearn.utils.testing import assert_array_equal
+
+iris = load_iris()
+
+
+@pytest.mark.parametrize(
+    "est",
+    [QuantileTransformer()]
+)
+def test_missing_value_handling(est):
+    # check that the preprocessing method let pass nan
+    rng = np.random.RandomState(42)
+    X = iris.data.copy()
+    n_missing = 15
+    X[rng.randint(X.shape[0], size=n_missing),
+      rng.randint(X.shape[1], size=n_missing)] = np.nan
+    X_train, X_test = train_test_split(X)
+    # sanity check
+    assert not np.all(np.isnan(X_train), axis=0).any()
+    assert np.any(X_train, axis=0).all()
```

Comment:
Is this the right condition?

Reply:
We check that there is some Nan in each column.

Reply:
Ups yes there I forgot to check for NaN :)